examples/chat_type_filter.py

```diff
@@ -22,8 +22,8 @@
 @dp.message_handler(chat_type=[ChatType.PRIVATE, ChatType.CHANNEL])
 async def send_welcome(message: types.Message):
     """
-    This handler will be called when user sends `/start` or `/help` command
-    """
+ This handler will be called when user sends message in private chat or channel
+ """
```

Comment:
Please indent these lines properly
```suggestion
    This handler will be called when user sends message in private chat or channel
    """
```